implementations/rust/ockam/ockam_core/src/error/inner/mod.rs

```diff
@@ -79,12 +79,17 @@ impl ErrorData {
         #[allow(unused_mut)]
         let mut local: Vec<LocalPayloadEntry> = vec![];
 
-        #[cfg(all(feature = "std", feature = "error-traces"))]
+        #[cfg(all(feature = "std", feature = "backtrace"))]
         if trace_config::BACKTRACE_ENABLED.get() {
+            // This is called everytime an Error instance is created and takes up to 300ms
+            // on a fast CPU, which is way slower than expected from such type of operation.
+            // Therefore, this feature is disabled by default, but can be used.
+            // It's also possible to replace `Backtrace::new()` with `Backtrace::new_unresolved()`
```

Comment:
That's a good point. Why not do `new_unresolved` by default and resolve when we want to log the error?